tools/checkpoint_conversion/convert_gpt_oss_checkpoints.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2024 The KerasHub Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+A conversion script for gpt_oss checkpoints.
+
+This script downloads a gpt_oss model from the Hugging Face hub,
+converts it to the Keras format, and saves it as a Keras preset.
+
+Usage:
+python convert_gpt_oss_checkpoints.py --preset=gpt_oss_8x7b_en
+"""
+
+import os
+import traceback
+
+os.environ["KERAS_BACKEND"] = "torch"
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"  # Hide any CUDA devices
+
+import numpy as np
+import torch
+from absl import app
+from absl import flags
+from keras import ops  # noqa: E402
+from transformers import AutoModelForCausalLM  # noqa: E402
+from transformers import AutoTokenizer  # noqa: E402
+
+import keras_hub  # noqa: E402
+
+device = torch.device("cpu")
+# Force PyTorch to use CPU
+torch.set_default_device(device)
+
+PRESET_MAP = {
+    "gpt_oss_20b_en": "openai/gpt-oss-20b",
```

Comment:
Will there be any other presets added later? I saw multiple presets in the register preset file.